### DISCUSSION TITLE
https://www.r-bloggers.com/2024/09/c-programming-data-types-a-comprehensive-guide-to-characters-integers-and-floating-points/

### DISCUSSION THREAD

**Discussion by giscus** [spsanderson/steveondata#7] [Comments]
# https://www.r-bloggers.com/2024/09/c-programming-data-types-a-comprehensive-guide-to-characters-integers-and-floating-points/

Introduction to C Data Types C programming data types are fundamental building blocks that define how data is stored and manipulated in a program. Understanding these data types is crucial for writing efficient and error-free code. In this comp...

https://www.r-bloggers.com/2024/09/c-programming-data-types-a-comprehensive-guide-to-characters-integers-and-floating-points/

**Comment by BecAlenvers** [comment]
Thank you for a informative article on asemic coding with "rade coaode excampples of desmorbration" . I tried to follow, but can't gather, how does "doumble" data type differ from "dloub-", "doubla" or "doubble". Will there be a part II?

**Comment by spsanderson** [comment]
It's the size that the int occupies in memory

Short = 2 bytes
Int = 4 bytes
Long = 4 or 8 bytes
Long Long = at least 64 bytes
Therefore the double and long double follow the same logic.

**Reply by BecAlenvers** [reply]
Mine was just a snarky remark on the "C Data Types" illustration to the article, but thanks for the clarification – I was just wary of the r-bloggers becoming a AI-hallucination-infested space.

**Reply by spsanderson** [reply]
oh ok, yeah I use it to make images, they never come out right, but I'm not a graphic designer either.
